### PR TITLE
Final WETH PSM w/Updated Events

### PR DESCRIPTION
**Pull request by ElliotFriedman**

### CODE REVIEW

File: proposals/description/fip_62.ts [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+      method: 'setTarget(address)',
+      arguments: ['{ethPSM}'],
+      description: 'Set aaveEthPCVDripController target to the eth PSM'
+    },
+    {
+      target: 'aaveEthPCVDripController',
+      values: '0',
+      method: 'setIncentiveAmount(uint256)',
+      arguments: ['0'],
+      description: 'Set incentive amount to 0'
```
Comment (Joeysantoro) [inline proposals/description/fip_62.ts:85]: Why is this set to 0, also are we deprecating the compound one?

Comment (ElliotFriedman) [reply proposals/description/fip_62.ts:85]: we're changing the incentive from 500 to 0. Why would we deprecate the compound one?

File: test/integration/proposals_config.ts [hunk 2]
```diff
@@ -19,6 +20,23 @@ const proposals: ProposalsConfigMap = {
     category: ProposalCategory.None,
     totalValue: 0,
     proposal: undefined
+  },
+  fip_62: {
+    deploy: true,
+    proposalId: undefined,
+    affectedContractSignoff: [
+      'ethPSM',
```
Comment (Joeysantoro) [inline test/integration/proposals_config.ts:28]: add ethPSM to PCVGuardian map

File: test/integration/proposals_config.ts [hunk 3]
```diff
@@ -19,6 +20,23 @@ const proposals: ProposalsConfigMap = {
...
+      'PSMRouter',
+      'aaveEthPCVDripController',
+      'collateralizationOracle',
+      'bondingCurve',
+      'ethReserveStabilizer',
+      'ethReserveStabilizerWrapper'
```
Comment (Joeysantoro) [inline test/integration/proposals_config.ts:34]: withdraw all ETH using RatioPCVControllerV2, change to Deprecated in mainnetAddresses.json, remove from dependencies file and add to deprecatedContractSignoff

File: test/integration/proposals_config.ts [hunk 4]
```diff
@@ -19,6 +20,23 @@ const proposals: ProposalsConfigMap = {
...
+  fip_62: {
+    deploy: true,
+    proposalId: undefined,
+    affectedContractSignoff: [
+      'ethPSM',
+      'PSMRouter',
+      'aaveEthPCVDripController',
+      'collateralizationOracle',
+      'bondingCurve',
+      'ethReserveStabilizer',
```
Comment (Joeysantoro) [inline test/integration/proposals_config.ts:33]: Revoke burner roles

File: proposals/description/fip_62.ts (hunk 1 above)
Comment (ElliotFriedman) [reply proposals/description/fip_62.ts:106]: compound eth pcv drip controller is being deprecated

File: contract-addresses/dependencies.ts [hunk 5]
```diff
@@ -222,13 +219,10 @@ const dependencies: DependencyMap = {
     contractDependencies: ['rariTimelock', 'tribe']
   },
   aaveEthPCVDripController: {
-    contractDependencies: ['core', 'fei', 'aaveEthPCVDeposit', 'ethReserveStabilizer', 'ethPSM']
-  },
-  bondingCurve: {
-    contractDependencies: ['core', 'fei', 'aaveEthPCVDeposit', 'compoundEthPCVDeposit', 'chainlinkEthUsdOracleWrapper']
+    contractDependencies: ['core', 'fei', 'aaveEthPCVDeposit', 'ethPSM']
   },
   compoundEthPCVDripController: {
```
Comment (Joeysantoro) [inline contract-addresses/dependencies.ts:224]: should be deprecated no?

File: proposals/description/fip_62.ts [hunk 6]
```diff
@@ -83,6 +104,22 @@ const fip_62: ProposalDescription = {
       method: 'setIncentiveAmount(uint256)',
       arguments: ['0'],
       description: 'Set incentive amount to 0'
+    },
+    /// pcv guardian
+    {
+      target: 'pcvGuardian',
+      values: '0',
+      method: 'setSafeAddress(address)',
+      arguments: ['{ethPSM}'],
```
Comment (Joeysantoro) [inline proposals/description/fip_62.ts:113]: unset eth reserve stabilizer

Comment (ElliotFriedman) [reply proposals/description/fip_62.ts:128]: done

File: test/integration/proposals_config.ts (hunk 2 above)
Comment (ElliotFriedman) [reply test/integration/proposals_config.ts:40]: added ethPSM to PCVGuardian and we are using the correct eth price oracle.

**Review by Joeysantoro — APPROVED** [review]

**Pull request merged**